pkgs/build-support/rust/default.nix

```diff
@@ -29,6 +29,7 @@
 , target ? null
 , cargoVendorDir ? null
 , checkType ? buildType
+, singleTestThread ? false
```

Comment:
How about?

```
enableParallelTesting` ? true
```

This seems a bit more consistent with `enableParallelBuilding`.

Reply:
There is an `enableParallelChecking` option used in a couple of places but I thought it might be ambiguous in this case as we would only be using it to set `test-threads=1` and not `cargo test -j 1` as well.

Reply:
Ok `enableParallelChecking` would be even better than. In this case we don't even need to introduce a new flag. I don't think it is `ambiguous` because the semantic is the same and it is ether `make check` or `cargo test` that gets executed but never both at the same time. Also having the `enable*` prefix makes it consistent with other options that we have in stdenv, which makes it easier to identify as a reader that this is something that influence the builder (similiar to `dont*`)

Reply:
Sorry but I think using `enableParallelChecking` is really unclear when all we are doing is setting the `rust` specific `test-theads` option.